### PR TITLE
#874 Start and clean services for running all frontend e2e tests

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #874 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/utils/apiHandler.js [hunk 1]
```diff
@@ -112,6 +133,38 @@ function sleep(milliseconds) {
...
+      serviceType: 'brokers',
+    };
+    let data = JSON.stringify([zkjson, bkjson]);
+    fs.writeFile('scripts/servicesApi/service.json', data, error => {
+      error;
+    });
+  });
+}
+
+async function jsonLoop(jsons, key, fun, configurator) {
```
Comment (konekoya) [inline ohara-manager/utils/apiHandler.js]: `fn`

File: ohara-manager/utils/apiHandler.js [hunk 2]
```diff
@@ -112,6 +133,38 @@ function sleep(milliseconds) {
   }
 }
 
+function fileHelper(zkName, bkName) {
+  fs.access('scripts/servicesApi/service.json', function(err) {
+    if (err) {
+      fs.mkdirSync('scripts/servicesApi');
+    }
+    let zkjson = {
```
Comment (konekoya) [inline ohara-manager/utils/apiHandler.js]: `const`

**Comment by konekoya** [comment]
Please address the above suggestions and merge the PR. Thanks :)

File: ohara-manager/utils/apiHandler.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/utils/apiHandler.js]: ok thx~

File: ohara-manager/utils/apiHandler.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/utils/apiHandler.js]: ok thx~

File: ohara-manager/utils/apiHandler.js [hunk 3]
```diff
@@ -53,35 +53,56 @@ async function createBk(configurator, port, zkName, bkName, nodeHost) {
   });
 }
 
-async function cleanNode(configurator, port, nodeHost) {
-  await axios.delete(`http://${configurator}:${port}/v0/nodes/${nodeHost}`);
+async function getApi(configurator, api, name) {
+  var res = await axios.get(`${configurator}/${api}`);
+  var result = res.data.some(e => e.name == name);
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply ohara-manager/utils/apiHandler.js]: ok thx~

File: ohara-manager/utils/apiHandler.js [hunk 4]
```diff
@@ -15,25 +15,25 @@
...
 async function createNode(
   configurator,
-  port,
   nodeHost,
   nodePort,
   nodeUser,
   nodePass,
 ) {
-  await axios.post(`http://${configurator}:${port}/v0/nodes`, {
+  await axios.post(`${configurator}/nodes`, {
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply ohara-manager/utils/apiHandler.js]: ok thx~

File: ohara-manager/scripts/cleanAllServices.js [hunk 5]
```diff
@@ -16,29 +16,16 @@
...
 
-const { configurator, port, nodeHost } = yargs.argv;
-
-debug('configurator: ', configurator);
-debug('port: ', port);
-debug('nodeHost: ', nodeHost || 'Not input.');
-
-function debug(...message) {
-  console.log(...message);
+async function start(configurator, nodeHost) {
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply ohara-manager/scripts/cleanAllServices.js]: ok thx~

File: ohara-manager/scripts/cleanAllServices.js [hunk 6]
```diff
@@ -16,29 +16,16 @@
...
-  var zk = JSON.parse(file).zk;
-  await api.cleanBk(configurator, port, bk);
-  await api.waitDelete(configurator, port, 'brokers');
-  await api.cleanZk(configurator, port, zk);
-  await api.waitDelete(configurator, port, 'zookeepers');
-  await api.cleanNode(configurator, port, nodeHost);
-};
-
-cleanServices();
+module.exports = { start };
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply ohara-manager/scripts/cleanAllServices.js]: ok thx~

File: .gitignore [hunk 7]
```diff
@@ -8,4 +8,6 @@ target/
 .DS_Store
 build
 out
-ohara-common/src/main/java/com/island/ohara/common/util/VersionUtils.java
\ No newline at end of file
+ohara-common/src/main/java/com/island/ohara/common/util/VersionUtils.java
+ohara-manager/client/scripts/servicesApi/service.json
```
[the remark this answers is not in the document]
Comment (wu87988622) [reply .gitignore]: ok thx~

**Pull request merged**